content/docs/projects/trust-manager.md

```diff
@@ -3,44 +3,58 @@ title: trust-manager
 description: 'Distributing Trust Bundles in Kubernetes'
 ---
 
-## Distributing Trust Bundles in Kubernetes
+trust-manager is the easiest way to manage TLS trust bundles in Kubernetes and OpenShift clusters.
```

Comment:
`TLS trust bundles` these trusted `X.509` certs are not only for TLS right, you could also use them for document signing etc.?

Reply:
Yeah that's true - I'm trying to get the keyword in there because "trust bundles" feels incredibly abstract and the number of people who'll use this for anything other than TLS is probably a rounding error.

X.509 is definitely (currently) more accurate... but maybe I'll just change to "trust bundles" for now so we don't have to change it later if we add e.g. JWK support